### PR TITLE
remove mutex on connector handler

### PR DESCRIPTION
**Pull request by ozgrakkurt**

### CODE REVIEW

File: packages/yoroi-extension/chrome/extension/background.js [hunk 1]
```diff
@@ -77,7 +77,6 @@ import type { lf$Database, } from 'lovefield';
 import { schema } from 'lovefield';
 import { copyDbToMemory, loadLovefieldDB, } from '../../app/api/ada/lib/storage/database/index';
 import { migrateNoRefresh } from '../../app/api/common/migration';
-import { Mutex, } from 'async-mutex';
```
Comment (zuzunker) [inline packages/yoroi-extension/chrome/extension/background.js:80]: @ozgrakkurt , you are removing the import for `Mutex`, but the mutex is still being created in the module on line 204

Comment (ozgrakkurt) [reply packages/yoroi-extension/chrome/extension/background.js:80]: @vsubhuman removed usage :+1:

**Review by yushih — APPROVED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**